nlp/gender_debiasing_cda/gender_debiasing_cda.ipynb

```diff
@@ -0,0 +1,1032 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ncXWQvfOH1CI"
+      },
+      "source": [
+        "# 🔭 Gender Bias in Word Embeddings 👫\n",
+        "\n",
+        "This notebook focuses on the existence of gender stereotypes in datasets and how to mitigate it through clever NLP!\n",
+        "\n",
+        "We'll train a word2vec model on the WikiBio dataset and look into the gender bias that exists in the learned vectors. Then, we'll apply Counterfactual Data Augmentation (CDA) method in our data to reduce bias and train a word2vec model on the new data.\n",
+        "\n",
+        "We hope that the new learned vectors will present less gender bias 😎.\n",
+        "\n",
+        "Of course, this is just one simple technique to try and mitigate (gender) bias in documents. There exist many more!"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "nkoNfTSd21g6"
+      },
+      "source": [
+        "## 🛠️ Getting started\n",
+        "\n",
+        "The cells below will configure everything that is required to get started with data loading using HuggingFace and training word2vec models using Gensim."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LQXfI15jQb1C"
+      },
+      "source": [
+        "### Setup"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "J8uFMXmmHzUY"
+      },
+      "outputs": [],
+      "source": [
+        "!apt-get install python3-magic\n",
+        "!pip install -q gensim==4.1.2 datasets augly gender-bender"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BTGe29mcQe-7"
+      },
+      "source": [
+        "### Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "id": "iLiKoNuq1MLn"
+      },
+      "outputs": [],
+      "source": [
+        "import numpy as np\n",
+        "from scipy import spatial\n",
+        "import multiprocessing\n",
+        "from datetime import datetime\n",
+        "import pandas as pd\n",
+        "\n",
+        "import plotly.graph_objects as go\n",
+        "\n",
+        "import augly.text as textaugs\n",
+        "from gender_bender import gender_bend\n",
+        "from sklearn.decomposition import PCA\n",
+        "\n",
+        "from datasets import concatenate_datasets, Dataset, load_from_disk"
```

Comment:
Would rearrange the order of package imports for better readability

Reply:
What would you change specifically? I now grouped the bottom 3 ones to fully adhere to the pylint paradigm of:
- first python default libs
- next 3rd party libs

and rearranged each group for ascending length